.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,249 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: ${{ github.event.comment.body == '/buildv1.20.14' }}
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/expressions#contains

use the contains function to trigger workflow

Reply:
The command should look like this：
/imagebuild version arch：/imagebuild v1.20.14 arm64

Reply:
> https://docs.github.com/en/actions/learn-github-actions/expressions#contains
> 
> use the contains function to trigger workflow

like this: startswith(github.event.comment.body, '/imagebuild')